binaries/go-http-completion/main.go

```diff
@@ -21,10 +21,33 @@ func completeProfiles(profilePrefix string) {
 	}
 }
 
+func completeRequests(partialName string, profileName string) {
+	requestNames, err := profile.GetAvailableRequests(profileName)
+	if err != nil {
+		panic(err)
+	}
+
+	for _, requestName := range requestNames {
+		if strings.HasPrefix(requestName, partialName) {
+			fmt.Printf("@%s\n", requestName)
+		}
+	}
+}
+
 func main() {
 	testString := os.Args[2]
 
 	if strings.HasPrefix(testString, "+") {
 		completeProfiles(testString[1:])
+	} else if strings.HasPrefix(testString, "@") || strings.HasPrefix(testString, "\\@") {
```

Comment:
Nit, it might be slightly more readable to move these booleans to their own helper functions (especially, since a couple are reused) like
```golang
strings.HasPrefix(value, "\\@")
```